db/index.js

```diff
@@ -11,6 +11,11 @@ const retrieve = (id, callback) => {
         if (err2) {
           callback(err2, null);
         } else {
+          for (let i = 0; i < data.length; i += 1) {
+            let text = data[i].text.split('\n');
+            console.log(text);
```

Comment:
I think you dont need the console.log anymore right?

Reply:
You are right I will remove. Thanks